back/spec/uploaders/base_image_uploader_spec.rb

```diff
@@ -28,7 +29,113 @@
       uploader.store!(file)
 
       image = MiniMagick::Image.open(uploader.file.path)
-      expect(image.exif).to be_empty
+
+      allowed_exif_keys = %w[
+        Orientation
+        ResolutionUnit
+        XResolution
+        YCbCrPositioning
+        YResolution
+      ]
+      expect(image.exif.keys - allowed_exif_keys).to be_empty
+
+      command = 'exiftool'
+      args = [uploader.file.path]
+      stdout_str, stderr_str, status = Open3.capture3(command, *args)
+      expect(status.success?).to(be(true), "exiftool command failed: #{stderr_str}")
+      exiftool_output = stdout_str # Now, exiftool_output holds the stdout
```

Comment:
```suggestion
      exiftool_output, stderr_str, status = Open3.capture3('exiftool', uploader.file.path)
      expect(status).to be_success, "exiftool command failed: #{stderr_str}"
```

I didn't know about custom messages. Just learned something new :)